civis_jupyter_notebooks/assets/initialize-git

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# NOTE: Similar logic exists in https://github.com/civisanalytics/replicator/blob/master/git-clone.sh
```

Comment:
What do you think about linking via a commit hash here instead of `master`? The implementation could change in the future, or this file could no longer exist in master.

Reply:
@rdeese  Does it make sense to use this one? It's the OG 😎 

```https://github.com/civisanalytics/replicator/commit/3691bec95295280cf03d7c864e531c60c19243dc```